golang/internal/exponential_backoff_test.go

```diff
@@ -0,0 +1,41 @@
+package internal
+
+import (
+	"testing"
+	"time"
+)
+
+func TestGetBackoffTime(t *testing.T) {
+	for i := int64(0); i < 1024; i++ {
+		max := 10 * time.Second
+		btime := GetBackoffTime(i, 1*time.Nanosecond, max)
+		if btime < 0 || btime > max {
+			t.Logf("BTime: %s", btime)
+			t.Logf("I: %d", i)
+			t.FailNow()
+		}
+	}
+}
+
+func TestPlotBackoffTime(t *testing.T) {
```

Comment:
should probably remove it

Reply:
Resolved in https://github.com/united-manufacturing-hub/united-manufacturing-hub/pull/690/commits/ba6e7cf265033bfe0530b893eac8bc5d73ae0062